modules/rosapps/applications/sysutils/screenshot/screenshot.c

```diff
@@ -273,13 +273,13 @@ ConvertDDBtoDIB(PSCREENSHOT pScrSht)
 	else cClrBits = 32;
 
 	if (cClrBits != 24)
-        pScrSht->lpbi = (PBITMAPINFO) HeapAlloc(GetProcessHeap(),
-                                                0,
-                                                sizeof(BITMAPINFOHEADER) + sizeof(RGBQUAD) * (1 << cClrBits));
-    else
-        pScrSht->lpbi = (PBITMAPINFO) HeapAlloc(GetProcessHeap(),
-                                                0,
-                                                sizeof(BITMAPINFOHEADER));
+    pScrSht->lpbi = (PBITMAPINFO) HeapAlloc(GetProcessHeap(),
```

Comment:
since the if blocks are more than one line, can you enclose them within brackes?